feat(react-native): send device and os info from the SDK (#690)
GetStream/stream-video-js: packages/react-native-sdk/docusaurus/docs/reactnative/01-basics/03-installation.mdx

- `stream-video-react-native` (SVRN) is Stream's Video SDK which contains UI components, hooks and util functions that will enable audio/video calls.
- `react-native-webrtc` is a WebRTC module for React Native, SVRN depends on this dependency, it's components and utilities to render audio/video tracks and interact with the phone's media devices.

> - `react-native-device-info` is responsible for get the device info in which the SDK is used and send it to the client.

Reviewer: ```suggestion
- `react-native-device-info` is responsible for getting the device info and sending it to the stream video client.
```
